### ISSUE TITLE
Improve landing page

### ISSUE THREAD

**Issue by ktjnyc** [hackforla/food-oasis#862]
### Overview
As a food seeker I need a simple landing page so that I can find food.  

### Action Items

- [x] Document the points of friction in a comment below (eg having to click the magnifying glass after entering address) #892 
- [x] Create Simple mock up of landing page that solves existing points of friction 
- [x] Mock up multiple versions of the landing page (with the option to use current location, with a button to re-use the location if you have said no)
- [x] Discuss with team and get approval on directions
- [x] Implement
  - [x] Landing page form fields have sensible defaults, location is not requested unless user initiates, and link to 'about this site' #940 
  - [x] Landing page background image changed to map image #939 
- [ ] Determine potential options for usability study (Optional)
- [ ] Draft message regarding why we are doing the usability study  (Optional)
- [ ] Create the usability study link (Optional)
- [ ] Send to Olivia to @channel for general slack (Optional)

### Resources/Instructions
[figma link](https://www.figma.com/file/D3oq9QOXl0rFkwrEUJbABs/Fola-Design-System?node-id=2743%3A7824)
Update this with usability hub link
[Link to document with usability items](https://docs.google.com/document/d/1Oiyk6OclfEMSee5pTxLWWOahmjaZH0QoZd_HisOA9us/edit)
[Link to 2021 UX study](https://drive.google.com/file/d/1FyIWm4SGqgopm_LESiChPoWFDP_TI2Z_/view?usp=sharing)

### Resolution
Option 3 (remove landing page altogether) is a major change for both UI and on the programming side and should be considered more deeply before attempting, and not sure that would be an actual improvement, so stopping that direction for now.

We've decided to just do:

Option 1 (improve form fields on landing page, and add link to learn more) and
Option 2 (change backgrounds to lower-bandwidth images of service area maps) for now.

**Comment by fancyham** [comment]
I had similar thoughts.

I created a prototype with some ideas for folks to play with and comment on in the figma document

@entrotech
See the slack for links to the prototype

**Comment by ktjnyc** [comment]
@fancyham I updated link to point to your document on usability and also added a link to the figma files here

**Comment by fancyham** [comment]
Decision in today's meeting to go with Option C) Remove landing page and instead jump directly to map, but with more prominent search field, "get current location" button, and "About this site" link as shown in the Figma prototype/designs.

**Comment by ktjnyc** [comment]
Bring over comments from #762:

Details on LA's free phone plan:
Free phone service in LA:
8GB SIM Only Data Plan
8GB LTE data,
Unlimited talk,
Unlimited global messaging every month.
6GB Standard Smartphone Data Plan
6GB LTE data,
FREE Smartphone*
Free Unlimited Talk and Text every month

While some Lifeline plans are generous with their data, many are not. Also, many folks don't have Lifeline plans and are paying out of pocket which means that they may opt for lower-data plans.

Lifeline phone service details:

"IMPORTANT NOTE: The plans of the various companies below can vary considerably. Some may give unlimited minutes, some unlimited texting, and most now include up or 500 MB of mobile internet data (or more!). And the phone models can vary greatly as well, with some providing smartphones. We urge you to compare plans of all providers who serve your state."

https://www.obamaphone.com/obamaphone-providers

500MB of data is nothing nowadays. Many web pages are 2+ MB, so that's less than 250 web pages.
https://www.google.com/search?q=average+web+page+mb

(our foodoasis.la initial map page is currently 1MB)

**Comment by entrotech** [comment]
We can create an issue or two for developers to work on implementing Option C, or continue the work on this issue. It might be a bit easier to do the changes somewhat incrementally:
1. Add the magnifier icon, Current Locaion Button and "Learn about..." link to the existing landing page.
2. Switch to a static map background image
3. Switch to skipping the landing page entirely.

While somebody is working on #1 and #2, someone else can be working on other issues with the Map page Issue #923, #926, #927, #929

**Comment by fancyham** [comment]
Can I suggest we change the title of this issue to something like “Improve arrival experience” or “Improve landing page” or merge it with an overarching issue as “Create Mock up of landing page” seems out of date now?

**Comment by ktjnyc** [comment]
#940

**Comment by fancyham** [comment]
If anyone is curious about California lifeline phones, check out this link.  The comments are great - gruff but helpful. Also gives you a sense of the issues people have with their phones.

TL;DR: One per household no matter how many in household, most have limited data with many in 3-5GB range, all are smartphones now.  

https://www.freegovernmentcellphones.net/states/california-government-cell-phone-providers

thanks for the link, @wanderingspoon

**Comment by entrotech** [comment]
Issue #940 implemented the changes to functionality on the landing page.
Issue #939 is not yet done, but would make the changes we agreed upon to the landing page baground image.
To get to Option 1C of Bryan's prototype, we would eliminate the landing page altogether, but I have not yet created a development issue for that until we get some feedback on the current implementation and can decide how to decide about eliminating the landing page.

**Comment by fancyham** [comment]
March 2 comments from cindywongg
 commented on Mar 2
Points of friction:

- clicking the magnifying glass after entering the address on the landing page
- landing page takes a while to load due to the high resolution background images
- whether or not user will understand the purpose of the website if the user is taken straight to the map page

https://github.com/hackforla/food-oasis/issues/868#issuecomment-789318990

**Comment by fancyham** [comment]
Option 3 (remove landing page altogether) is a major change for both UI and on the programming side and should be considered more deeply before attempting, and not sure that would be an actual improvement, so stopping that direction for now.

We've decided to just do:
- Option 1 (improve form fields on landing page, and add link to learn more) and 
- Option 2 (change backgrounds to lower-bandwidth images of service area maps) for now.
